### PR TITLE
Security

### PR DESCRIPTION
**Pull request by heshambadawy**

### CODE REVIEW

File: src/app/core/data.api/auth.service.ts [hunk 1]
```diff
@@ -25,7 +24,7 @@ export class AuthenticationService {
 
   login(user: AuthenticatedUser) {
     if (user && user.accessToken) {
-      this.cookieService.set('current_user', JSON.stringify(user), undefined);
+      this.cookieService.set('current_user', JSON.stringify(user), 365);
```
Comment (Aychihuahuausa) [inline src/app/core/data.api/auth.service.ts:27]: what is 365

File: src/app/core/data.api/auth.service.ts [hunk 2]
```diff
@@ -41,12 +40,12 @@ export class AuthenticationService {
   }
 
 
-  validateUser() {
+  validateUser(isGuard =false) {
     console.log('\x1b[33m%s\x1b[0m', this.currentUserValue)
```
Comment (Aychihuahuausa) [inline src/app/core/data.api/auth.service.ts:44]: remove console log

File: src/app/core/interceptors/error-interceptor.ts [hunk 3]
```diff
@@ -1,39 +1,45 @@
...
                                 ) ,
-                                catchError((err , obs)=> {tries++ ;   return obs})
+                            finalize(()=>{console.log('TRYING TO UPDATE TOKEN HAS BEEN FINALIZED')})
                         )
                 }
                 else {
-                    console.log('\x1b[33m%s\x1b[0m', tries) 
-                    console.log(this.authenticationService.currentUserValue)
+                    console.log('\x1b[33m%s\x1b[0m',  window.tries)
+                    console.log('\x1b[33m%s\x1b[0m', this.authenticationService.currentUserValue)
```
Comment (Aychihuahuausa) [inline src/app/core/interceptors/error-interceptor.ts:42]: remove log

File: src/app/core/data.api/auth.service.ts (hunk 2 above)
Comment (heshambadawy) [reply src/app/core/data.api/auth.service.ts:44]: Same here Need that

File: src/app/core/data.api/auth.service.ts (hunk 1 above)
Comment (heshambadawy) [reply src/app/core/data.api/auth.service.ts:27]: it is 365 days after that the cookie will be expired

**Pull request merged**